src/scalardl/src/main/java/com/example/contract/token/Const.java

```diff
@@ -0,0 +1,23 @@
+package com.example.contract.token;
+
+public class Const {
+
+  private Const() {}
```

Comment:
Do we need this ?

Reply:
I removed it (also removed same from smallbank ones).